crates/dprint/src/arg_parser.rs

```diff
@@ -195,6 +197,19 @@ fn inner_parse_args<TStdInReader: StdInReader>(args: Vec<String>, std_in_reader:
     ("editor-service", matches) => SubCommand::EditorService(EditorServiceSubCommand {
       parent_pid: matches.get_one::<String>("parent-pid").and_then(|v| v.parse::<u32>().ok()).unwrap(),
     }),
+    #[cfg(feature = "completions_generator")]
+    ("generate-completions", matches) => {
+      let mut cmd = create_cli_parser(false);
+
+      clap_complete::generate(
+        matches.get_one::<clap_complete::Shell>("shell").unwrap().to_owned(),
+        &mut cmd,
+        "dprint",
+        &mut std::io::stdout(),
+      );
```

Comment:
Is it possible to have this be done below at?

https://github.com/dprint/dprint/pull/678/files#diff-50367ae714c483a503720af7c834d1c5297d348ee58b8a1878536cc007305874R136

Then we can reserve this function for just parsing and not functionality.